### PR TITLE
Avatar and badge

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (419)
Screenshot (420)
Screenshot (432)
Screenshot (433)

Added badge component and made changes in px to rem and also used css variables as suggested.

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    padding: 20px;
+    font-size: 19px;
+    color: var(--primary-color);
+}
+.avatar-container{
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+    align-items: center;
+    justify-content: center;
```
Comment (Hamdrive) [inline components/avatar/avatar.css:26]: Try to use helper classes, helps to avoid repitition

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    width: 50px;
+    height: 50px;
+}
+.normal{
+    width: 100px;
+    height: 100px;
+}
+.large{
+    width: 150px;
+    height: 150px;
```
Comment (Hamdrive) [inline components/avatar/avatar.css:53]: Use `rems` or atleast predefined sizes in `:root`
Also try to use abbreviations (sm, md , lg)

File: components/avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                <div class="avatar-container">
+                    <img src="../../assets/avatar-image.jpg" alt="" class="round-av">
+                </div>
+                <div class="avatar-type">
+                    Round Avatar with different sizes
+                </div>
+                <div class="avatar-container">
+                    <img src="../../assets/avatar-image.jpg" alt="" class="round-av small">
+                    <img src="../../assets/avatar-image.jpg" alt="" class="round-av normal">
+                    <img src="../../assets/avatar-image.jpg" alt="" class="round-av large">
```
Comment (Hamdrive) [inline components/avatar/avatar.html:80]: Try to use shorter abbreviated class names
`rd-av sm` would work for line 78

File: components/avatar/avatar.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                <div class="avatar-container">
+                    <img src="../../assets/avatar-image.jpg" alt="" class="squared-av">
+                </div>
+                <div class="avatar-type">
+                    Square Avatar with different sizes
+                </div>
+                <div class="avatar-container">
+                    <img src="../../assets/avatar-image.jpg" alt="" class="squared-av small">
+                    <img src="../../assets/avatar-image.jpg" alt="" class="squared-av normal">
+                    <img src="../../assets/avatar-image.jpg" alt="" class="squared-av large">
```
Comment (Hamdrive) [inline components/avatar/avatar.html:94]: Same point as above, use abbreviated classnames

**Review by Hamdrive — COMMENTED** [review]
Good work done
Have given some pointers to rectify to make it even better!

File: components/avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+.avatar-container{
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+    align-items: center;
+    justify-content: center;
+    padding: 30px;
+    background-color: #dbe0e7;
+    border-radius: 5px;
+    flex-wrap: wrap;
```
Comment (MaruthiPothuganti) [inline components/avatar/avatar.css:30]: Suggestion:  _flex-direction: row_ and  _flex-wrap: wrap;_ can be combined into _flex-flow: row wrap_

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended changes: Use of short-handed property.
Everything else looks good.

**Pull request merged**